west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: edf807fd47cd6c3ab71c020363888fe73f2cc776
+      revision: pull/1324/head
```

Comment:
Please update SHA: `fe211cd2a757b6fea4122bb7791021f100bce7cd`

Reply:
Sorry it took me so long, got meeting to attend.